templates/index.html

```diff
@@ -1,10 +1,13 @@
-$def with (output, now)
+$def with (output, now, download_api)
 
 <body onload="draw($output[0]['s_rate']);">
 $ errors = 0
 $ color = "green"
 <div>
-<div style="height: 40%; overflow-y: scroll; overflow-x: hidden;">
+<div class="download-header">
+  <button class="download-button" onclick="window.location.href = '$download_api';">Download Report</button>
+</div>
+<div style="height: 38%; overflow-y: scroll; overflow-x: hidden;">
```

Comment:
Maybe a link on the footer (please see https://github.com/adriaaah/monit-dashboard/pull/11/files#r352238464) would suffice instead of having a permanent banner in the main area

Reply:
@adriaaah I would like to highlight this `download`, however as you suggested I am going to change the view from horizontal to vertical (https://github.com/adriaaah/monit-dashboard/issues/10) soon and will think some better solution for this `download` button on vertical view.